src/components/monitor/AllGoodSection.tsx

```diff
@@ -0,0 +1,15 @@
+import { CheckCircle, Smiley } from '@phosphor-icons/react/dist/ssr';
+
+export const AllGoodSection = ({ textContent }) => {
```

Comment:
Type the props. Use an interface and then use `HaveIbeenPawnedText[...]`

Reply:
Done!